relay-dynamic-config/src/defaults.rs

```diff
@@ -270,6 +270,23 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
                     .when(is_http.clone()),
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/exclusive_time@millisecond".into(),
```

Comment:
The intention is to emit the same metric with less tags for all spans but still have the additional tags on the metric for certain conditions.

I suppose I'm expecting Relay to emit both metrics when conditions are met for both but then aggregate the 2 before it's sent.

Reply:
Actually, since it'll emit the count twice, that doesn't work, I need to do `condition` and `!condition`.

Reply:
IMO it would be more declarative to update the condition on the existing metric to `None` (i.e. always extract), and then change the `.always()` calls on the tags to `.when(exclusive_time_condition.clone()`.

We might want to rename `exclusive_time_condition` to `known_modules` or something similar.